src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -0,0 +1,372 @@
+              aria-label={`Previous ${this.itemLabel.toLocaleLowerCase()}`}
+              disabled={this.selectedIndex < 1}
+              mode="ghost"
+              onClick={() => {
+                this.selectedIndex -= 1
+              }}
+              part="arrow prev focusable"
+              size={this.size}
+            >
+              <ld-icon name="arrow-left" size={this.size} />
+            </ld-button>
+          </li>
+          {this.renderSticky &&
+            Array.from({ length: Math.min(this.sticky, this.length) }).map(
+              (_: unknown, index: number) => {
+                return (
+                  <li class="ld-pagination__sticky">
+                    <ld-button
+                      aria-label={`${this.itemLabel} ${index + 1}`}
+                      mode="ghost"
+                      onClick={() => {
+                        this.selectedIndex = index
+                      }}
+                      part="sticky item focusable"
+                      size={this.size}
+                    >
+                      {index + 1}
+                    </ld-button>
+                  </li>
+                )
+              }
+            )}
+          {this.renderDots && (
+            <li
+              class={getClassNames([
+                'ld-pagination__dots',
+                showStartDots && 'ld-pagination__dots--visible',
+              ])}
+              part="dots"
+            >
+              <span>. . .</span>
+            </li>
+          )}
+          <li
+            class="ld-pagination__slide-wrapper"
+            part="slide-wrapper"
+            style={{
+              '--ld-pagination-slider-cols': `${Math.min(
+                this.slidableItems.length,
+                this.maxSliderColumns
+              )}`,
+            }}
+          >
+            <ul
+              class="ld-pagination__items"
+              part="items"
+              style={{
+                '--ld-pagination-slide-index': `${Math.max(
+                  Math.min(
+                    this.selectedIndex - this.offset - 1,
+                    // -1 because of the "..." and another -1 because it is 0-based
+                    this.length - this.visibleItemsInSlider - this.sticky - 2
+                  ),
+                  // marker position in case of length < sticky
+                  Math.min(this.sticky, this.length),
+                  0
+                )}`,
+              }}
+            >
+              <li
+                class="ld-pagination__marker"
+                key="marker"
+                part="marker"
+                style={{
+                  '--ld-pagination-selected-index': `${this.selectedIndex}`,
+                }}
+              />
+              {this.length > 0 &&
+                this.sliderContent.map((itemNumber) =>
+                  this.renderItem(itemNumber, showFrom, showTo)
+                )}
+            </ul>
+          </li>
+          {this.renderDots && (
```

Comment:
I thought the dots feature has not been implemented yet? Just make sure we do not ship half baked features 😉

Reply:
It has and is fully functional.

Reply:
No, I don't think so. Seems like the `mode` prop doesn't exist at all.

Reply:
This is about the "more" dots ("..."), not about the dots mode. Renaming it to prevent confusion.